CHANGELOG.md

```diff
@@ -51,6 +51,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Display not scrolling when printing new lines while scrolled in history
 - Regression in font rendering on macOS
 - Scroll down escape (`CSI Ps T`) incorrectly pulling lines from history
+- Dim truecolor foreground
```

Comment:
This almost makes it sound like the truecolor foreground is now always dimmed, I think we can be a little more specific:

```suggestion
- Dim escape (`CSI 2 m`) support for truecolor text
```

Reply:
Shouldn't that be in `Added` section or it's fine to be in `Fixed`?

Reply:
I think fixed is fine, since it's kind of a bug that it hasn't been done until now.